spec/haskell/src/SEL4/Object/Notification.lhs

```diff
@@ -131,6 +131,7 @@ If a notification object is deleted, then pending receive operations must be can
 
 > cancelAllSignals :: PPtr Notification -> Kernel ()
 > cancelAllSignals ntfnPtr = do
+>         stateAssert ksReadyQueues_asrt "asserts several properties about ksReadyQueues"
```

Comment:
I still think these "asserts several properties ..." assertions are a bit noisy (don't provide any info that the `ksReadyQueues_asrt` already says), but it's probably not wise to get into that again, I think it's better than the last time I saw them

Reply:
Ah I did actually remove these comments in the `rt` version after you suggested it. I think there was some mixup with the commits at some stage and I lost some work on the `master` branch, which I then redid. Maybe this was part of it and I missed it. Still happy to just remove the comment entirely and have it as just `""` if that's preferable

Reply:
yeah, if you already did that and lost it, maybe revive it. cheers